substrate/frame/im-online/src/benchmarking.rs

```diff
@@ -19,17 +19,15 @@
 
 #![cfg(feature = "runtime-benchmarks")]
 
-use super::*;
-
-use frame_benchmarking::v1::benchmarks;
+use frame_benchmarking::{v2::*, BenchmarkError};
```

Comment:
```suggestion
use frame_benchmarking::v2::*;
```

`BenchmarkError` should already be in v2